packages/dashboards/tsconfig.json

```diff
@@ -17,7 +17,8 @@
         "paths": {
             "@nova-ui/dashboards": [
                 "./src/public-api.ts"
-            ]
+            ],
+            "components/*": ["./examples/src/components/*"]
```

Comment:
Because of an error in eslint "Unable to resolve path to module 'components/prototypes/data/table/constants'  import/no-unresolved".
Moved this fix into "examples" `.eslintrc.json` file